### PR TITLE
Added topic-changed-event on triggers.py

### PR DESCRIPTION
**Pull request by erenatas**
I have added missing attribute "topic-changed-event" and also written test.

### CODE REVIEW

File: jenkins_jobs/modules/triggers.py [hunk 1]
```diff
@@ -349,6 +350,8 @@ def gerrit(registry, xml_parent, data):
          * **draft-published-event** -- Trigger on draft published event.
          * **ref-updated-event** -- Trigger on ref-updated.
            Gerrit Trigger Plugin version >= 2.29.0
+         * **topic-changed-event** -- Trigger on topic-changed.
+           Gerrit Trigger Plugin version >= 2.30.5
```
Comment (romanek-adam) [inline jenkins_jobs/modules/triggers.py]: According to [GitHub commit](https://github.com/jenkinsci/gerrit-trigger-plugin/commit/32d6e971e6c6a89d4376bbd421622af766d3aaa4) the implementation was added in v2.26.0.

File: tests/triggers/fixtures/gerrit028-trigger-simple-triggers.plugins_info.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- longName: 'Gerrit Trigger'
```
Comment (romanek-adam) [inline tests/triggers/fixtures/gerrit028-trigger-simple-triggers.plugins_info.yaml]: this file is only needed when targeting an older version of the plugin that the most recent one supported by JJB

File: jenkins_jobs/modules/triggers.py [hunk 3]
```diff
@@ -155,6 +155,7 @@ def build_gerrit_triggers(xml_parent, data):
         "patchset-uploaded-event": "PluginPatchsetCreatedEvent",
         "patchset-created-event": "PluginPatchsetCreatedEvent",
         "ref-updated-event": "PluginRefUpdatedEvent",
+        "topic-changed-event": "PluginTopicChangedEvent",
```
Comment (romanek-adam) [inline jenkins_jobs/modules/triggers.py:159]: Looks like the keys were meant to be ordered alphabetically, could you please fix that? (and also the order of "private-state-changed-event")

**Comment by romanek-adam** [comment]
Since we don't use this functionality (I assume so) then we can wait with merge until the change gets approved upstream (to avoid diff between repos).

Have you submitted this change to the upstream repo for review?

**Comment by erenatas** [comment]
> Since we don't use this functionality (I assume so) then we can wait with merge until the change gets approved upstream (to avoid diff between repos).
> 
> Have you submitted this change to the upstream repo for review?

No I haven't but I will.

**Review by romanek-adam — APPROVED** [review]

**Review by romanek-adam — APPROVED** [review]

**Review by vdanaila — APPROVED** [review]

**Pull request merged**